packages/client/components/ReflectionGroup/ReflectionGroup.tsx

```diff
@@ -205,7 +205,7 @@ const ReflectionGroup = (props: Props) => {
         />
       )}
       <Group
-        {...(isDroppable ? {[DragAttribute.DROPPABLE]: reflectionGroupId} : null)}
+        {...(isBehindSpotlight ? null : {[DragAttribute.DROPPABLE]: reflectionGroupId})}
```

Comment:
Temporary fix. Will be solved by another PR. See discussion here https://github.com/ParabolInc/parabol/pull/5600#discussion_r751129213